Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
Now we are both confused? ;-) I suggest the following:
- Remove -race from test/test-verbose
- Add test-full
   - Call test-verbose
   - Do a run with -race and -v
   - Do a run with -msan and -v
- Call test-full in the CI

That makes the normal testing process (test/test-verbose) fast, and the CI would catch everything no matter what.